tutorials/i18n/pseudolocalization.rst

```diff
@@ -0,0 +1,82 @@
+.. _doc_pseudolocalization:
+
+Pseudolocalization
+==================
+
+Introduction
+------------
+
+The process of localization usually starts when development has finished. This
+means that translations aren't available during development for testing out
+whether the project is internationalized properly. Godot offers pseudolocalization
+as a way to test how robust the project is when it comes to locale changes.
+Pseudolocalization simulates changes that might take place during localization so
+that any issues regarding Internationalization can be recognized early on during
+development.
+
+.. note:: The official demo for pseudolocalization can be used to play around with 
+          the various properties and observe their effects; you can
+          `download it from this GitHub repository <https://github.com/angad-k/Pseudolocalization-Demo>`_.
+
+
+Enabling and configuring pseudolocalization
+-------------------------------------------
+
+Enabling pseudolocalization and the configurations related to it is as simple as
+toggling a checkbox in the project settings. These settings can be found in
+**Project → Project Settings → General → Internationalization → Pseudolocalization**:
+
+.. image:: img/pseudolocalization_settings.png
+
+Pseudolocalization can also be toggled at the runtime through GDScript (more on that 
+at the end of this tutorial).
+
+Pseudolocalization configurations
+---------------------------------
+
+Pseudolocalization in Godot can be set up according to the specific use case of the 
+project. Following are the pseudolocalization properties that can be configured through 
+project settings :
+
+-  ``replace_with_accents``, replaces all characters in the string with their accented 
+    variants. "The quick brown fox jumped over the lazy dog" will be converted to 
+    "Ŧh̀é q́üíćḱ ḅŕôŵή f́ôx́ ǰüm̀ṕéd́ ôṽéŕ ŧh̀é łáźý d́ôǵ" when this setting is enabled. 
+-  ``double_vowels``, doubles all the vowels in the string. It is a pretty good approximation
+    to simulate expansion of text during localization.
+-  ``fake_bidi``, or fake bidirectional text, simulates right-to-left text. This is useful to
+    simulate right-to-left writing systems.
+-  ``override``, replaces all the characters in the string with a '*'. This is useful for 
+    quickly finding text that isn't being localized.
+-  ``expansion_ratio``, can be used in cases where doubling the vowels isn't a sufficient 
+    approximation. This setting pads the string with '_' and expands it by the given ratio.
+    An expansion ratio of ``0.3`` is sufficient for most practical cases and it will increase 
+    the length of the string by 30%.
+-  ``prefix`` and ``suffix``, properties can be used to specify a prefix and suffix to wrap
+    the text in.
+-  ``skip_placeholders``, skips placeholders for string formatting like ``%s`` and ``%f``. 
+    This is useful to identify places where more arguments are required for the formatted string
+    to display correctly.
```

Comment:
```suggestion
- ``replace_with_accents``: Replaces all characters in the string with their accented 
   variants. *"The quick brown fox jumped over the lazy dog"* will be converted to 
   *"Ŧh̀é q́üíćḱ ḅŕôŵή f́ôx́ ǰüm̀ṕéd́ ôṽéŕ ŧh̀é łáźý d́ôǵ"* when this setting is enabled.
   This can be used to spot untranslated strings that won't have accents,
   but is also useful to check for missing glyphs in the font(s) used by the project.
- ``double_vowels``: Doubles all the vowels in the string. It is a good approximation
   to simulate expansion of text during localization.
   This can be used to check for text that would overflow its container
   (such as buttons).
- ``fake_bidi``: Fake bidirectional text (simulates right-to-left text). This is useful to
   simulate right-to-left writing systems to check for potential layout issues that would occur in languages using right-to-left scripts.
- ``override``: Replaces all the characters in the string with an asterisk (``*``). This is useful for 
   quickly finding text that isn't being localized.
- ``expansion_ratio``: Can be used in cases where doubling the vowels isn't a sufficient 
   approximation. This setting pads the string with underscores (``_``) and expands it by the given ratio.
   An expansion ratio of ``0.3`` is sufficient for most practical cases; it will increase 
   the length of the string by 30%.
- ``prefix`` and ``suffix``: These properties can be used to specify a prefix and suffix to wrap
   the text in.
- ``skip_placeholders``: Skips placeholders for string formatting like ``%s`` and ``%f``. 
   This is useful to identify places where more arguments are required for the formatted
   string to display correctly.
```